src/runtime/hexagon_dma.cpp

```diff
@@ -6,6 +6,22 @@
 #include "mini_hexagon_dma.h"
 #include "hexagon_dma_pool.h"
 
+#define ASSERT_FORMAT(user_context) {\
```

Comment:
Hmm thought I commented on this. I think this should be a function to convert eDmaFmt to halide_hexagon_dma_format (or whatever), rather than just asserting the enums have the same value. If you want to implement the conversion function by casting and asserting there, I guess that seems OK (but I think just a big switch statement would be better), but it shouldn't be a macro like this.